bin/nrnivmodl.in

```diff
@@ -3,31 +3,34 @@ set -e
 
 ARCH=@host_cpu@
 MODSUBDIR=@modsubdir@
-USER_PWD=`pwd`
+USER_PWD="`pwd`"
+
+UsingCMake="no"
+@USING_CMAKE_TRUE@UsingCMake="yes"
 
 if test "x${NRNHOME}" = x ; then
-	prefix=@prefix@
-	exec_prefix=@exec_prefix@
-	bindir=@bindir@
-	libdir=@libdir@
+  prefix=@prefix@
```

Comment:
nitpick : some insistent spacing / indentation. (typically we use 4. this script uses 2 or at some places 6)

Reply:
now freeing modprefix at end of main.
Added a comment about the fact that shell lines in make rules must be indented with tab.